### DISCUSSION TITLE
Display multiple projects on the same page

### DISCUSSION THREAD

**Discussion by iosifnicolae2** [community/community#46254] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Hi.

It would be great if we can have **multiple Github Projects on the same page** and to be able to filter the issues based on the project.

Also, this will reduce the number of clicks required to make in order to open a project, because instead of clicking on `Projects` and then click on a project, we can just click on `Projects` and we have all the configured projects displayed.

Thank you very much!

**Comment by nicosuave** [comment]
Secondly, would love to be able to stack rank issues across multiple different projects and to have this rank preserved!
